src/data/behandler/behandlereQueryHooks.ts

```diff
@@ -41,3 +42,28 @@ export const useSokBehandlereQuery = (searchstring: string) => {
     data: query.data || [],
   };
 };
+
+export const useBehandlerByBehandlerRefQuery = (
+  behandlerRef: string | false
+) => {
+  const behandlerRefHeader = { behandlerRef: behandlerRef ? behandlerRef : "" };
+  const fetchBehandlerByBehandlerRef = () =>
```

Comment:
Ikke superviktig, men siden `behandlerRef` er en slags uuid for en behandler tenker jeg det egentlig hadde vært mer naturlig om API-et i isdialogmelding var `${ISDIALOGMELDING_ROOT}/behandler/${behandlerRef};` Får evt se på det i egen PR.

Reply:
Ja, egentlig litt enig med deg 🤔